cli/src/pcluster/aws/ec2.py

```diff
@@ -312,8 +312,8 @@ def get_supported_az_for_instance_type(self, instance_type: str):
         """
         Return a tuple of availability zones that have the instance_type.
 
-        This function build above _get_supported_az_for_instance_types,
```

Comment:
I could not find a function called `_get_supported_az_for_instance_types`, but I saw that `get_supported_az_for_instance_types` (without the underscore at the beginning) is defined just below this function. The version without the initial underscore is the function that's actually called within `get_supported_az_for_instance_type`, so I assume this is the function that was actually meant in the comment

Reply:
correct